### DISCUSSION TITLE
Weird minor splash screen bug

### DISCUSSION THREAD

**Discussion by brianjz** [vladmandic/sdnext#1799] [General]
I didn't know the three orange-ish lines on the splash screen rotated until I completely refreshed my build from scratch to make sure I had no weird lingering issues (it's been awhile.)

After some debugging, it turns out if you disable "sd-webui-agent-scheduler", the orange lines do not rotate. Anyone else seeing this? I'm not sure it was by design.

This is on Firefox and I had the bug in both desktop and mobile.

**Comment by vladmandic** [comment]
definitely not by design, i'll check.

**Reply by brianjz** [reply]
Such a weird one. I thought maybe I had something cached or stuck on my end, but then after I started fresh and then set my config settings again, it stopped. Went through stuff a few lines at a time until I saw it stop again. Not sure if it affects anything else.

**Comment by vladmandic** [comment]
fixed.
